src/components/BooksList.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../containers/Book';
+
+function BooksList({ books }) {
+  const bookList = books.map(book => (
+    <Book book={book} key={Math.random()} />
```

Comment:
- [ ]  The BookList component it's connected with redux, therefore, should be placed at `src/containers`